vizro-core/src/vizro/actions/_filter_interaction.py

```diff
@@ -100,7 +100,7 @@ def outputs(self):
 
         for target in self.targets:
             component_id = target
-            component_property = cast(FigureType, model_manager[target])._output_component_property
+            component_property = cast(FigureType, model_manager[target])._output_default_property
```

Comment:
@huong-li-nguyen I _think_ one of the acceptance criteria for this should be that we can just do `return self.targets` as per this part of the TODO:

```
When this change is made the outputs property for filter, parameter and on_page_load should just
become `return self.targets` or similar.
```

Does that work with what you have here already?